doc/in-progress/sui-wallet-with-ledger.md

```diff
@@ -0,0 +1,86 @@
+1. Click **Connect Ledger Wallet**.
+   You might need to scroll to see the button if you have a lot of accounts in your wallet.
+1. Unlock your device before you perform the next step.
+1. On the **Connect Ledger Wallet** screen, click **Continue**.
+1. Select your device when prompted and then click **Connect**.
+1. When connected, choose the account or accounts to import to Sui Wallet, and then click **Unlock**.
+
+Sui Wallet displays the accounts you selected on the **Accounts** screen. The accounts imported from your Ledger device show *LEDGER* next to them. When you use Sui Wallet, you can select the account to use when you connect to a site or perform a transaction.
+
+##  View your account balance and assets
+
+You can view the balance for an account in Sui Wallet on the **Coins** tab. The coins and SUI stake displayed are for only the selected account address. To view the balance for a different account address, select the account from the drop-down list near the top of the wallet.
+
+To view the coins and tokens in your account you imported from your Ledger device, select an account that displays *LEDGER* next to it. To view the NFTs for the account, click the **NFTs** tab in Sui Wallet.
+
+## Receive digital assets using Sui Wallet
+
+To receive cryptocurrency or other digital assets such as NFTs in Sui Wallet, select the account to receive the assets on the **Coins** tab, then click the copy icon to copy the address.
+
+Before you use an account from your Ledger device to receive an asset, confirm that the account is connected to your device.
+
+**To receive digital assets to your Ledger account using Sui Wallet** 
+1. Open Sui Wallet and click the Settings menu (the 3 bars displayed in the top-right corner).
+1. Click **Accounts**.
+1. Click the down arrow next to the Ledger account to receive the asset, then click **Verify Ledger connection**. Make sure that the display shows **Ledger is connected** before you initiate the transaction to receive an asset.
+1. Send the digital asset to the address for the Ledger account.
+
+## Send digital assets using Sui Wallet
+
+You can use Sui Wallet to send a digital asset from your Ledger account.
+
+**To send digital assets from your Ledger account using Sui Wallet** 
+1. Open Sui Wallet and click the Settings menu (the 3 bars displayed in the top-right corner).
+1. Click **Accounts**.
+1. Click the down arrow next to the Ledger account to receive the asset, then click **Verify Ledger connection**. Make sure that the display shows **Ledger is connected** before you initiate the transaction to send an asset.
+1. Click the **Coins** tab in Sui Wallet, then click **Send**.
+Enter the amount of SUI to send and the address of the recipient, then click **Review**.
+1. Review the details of the transaction to confirm that they are accurate. If they are, click **Send now**.
+1. On your Ledger device, press both buttons to authorize the transaction **Transfer SUI**.
```

Comment:
Were you able to successfully send SUI from a Ledger account? There are a few prompts you have to go through in the Ledger device, I wasn't sure if we needed to include in detail how to navigate those prompts (e.g. viewing the from address, to address, amount being sent, gas price, hitting confirm, etc.)

Reply:
I was not as I could not complete a transaction to the Ledger account so that I had a balance to send back